src/main/java/io/vertx/spi/cluster/zookeeper/ZookeeperClusterManager.java

```diff
@@ -48,17 +52,19 @@
 
   private String nodeID;
   private CuratorFramework curator;
+  private boolean customCuratorCluster;
   private RetryPolicy retryPolicy;
   private Map<String, ZKLock> locks = new ConcurrentHashMap<>();
 
   private static final String DEFAULT_CONFIG_FILE = "default-zookeeper.properties";
   private static final String CONFIG_FILE = "zookeeper.properties";
-  private static final String ZK_SYS_CONFIG_KEY = "vertx.zookeeper.conf";
+  private static final String ZK_SYS_CONFIG_KEY = "vertx.zookeeper.config";
```

Comment:
Should be align with the other cluster manager
(just wondering if we have such a convention)